app/controllers/answers_controller.rb

```diff
@@ -9,7 +9,7 @@ def create
     if answer.save
       redirect_to :back
     else
-      redirect_to new_answer_path
+      render new_answer_path
```

Comment:
Eh, you need to render a _file_ not a route.  This won't work.

...and a test would help you verify it (HINT).